server/test/Endpoints/router-test.js

```diff
@@ -3,11 +3,13 @@ const request = require('supertest');
 const app = require('../../app');
 const dbBuild = require('../../database/config/db_build');
 
+
 test('Get all student information from /api/v1/student/', (t) => {
   dbBuild()
     .then(() => {
       request(app)
         .get('/api/v1/student/')
+        .set('Cookie', ['jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MTIzNDUsInJvbGUiOiJzdHVkZW50IiwiaWF0IjoxNTU2NjA2MjQ4fQ.jxmj_2S-sGc7XKoZPkh0JDDTR1AoWJvNggokoC54QP4'])
```

Comment:
@AbdallahAmmar96 You could move this cookie value into a const variable, and then if you ever have to change it in the future, you only have to change one line instead of many lines.